shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitParameters.java

```diff
@@ -67,4 +71,13 @@ public ShortCircuitParameters setSubTransStudy(boolean subTransStudy) {
         return this;
     }
 
+    public boolean isWithFeederContributions() {
```

Comment:
I though we decided to remove the term "contributions" from the API ?

Reply:
feederLevelResults ????